crates/rpc/rpc-eth-types/src/builder/config.rs

```diff
@@ -45,6 +73,8 @@ pub struct EthConfig {
     pub fee_history_cache: FeeHistoryCacheConfig,
     /// The maximum number of getproof calls that can be executed concurrently.
     pub proof_permits: usize,
+    /// The pending block config
+    pub pending_block: PendingBlockConfig,
```

Comment:
I'd like to name this pending_block_kind so we dont confuse this var with the actual pending_block var